.gitlab-ci.yml

```diff
@@ -245,6 +245,32 @@ build:base+async:
     COQ_EXTRA_CONF: "-native-compiler yes -coqide opt"
     COQUSERFLAGS: "-async-proofs on"
 
+quick:build+validate:
+  stage: test
+  script:
+    - set -e
+
+    - echo 'start:coq.config'
+    - ./configure -local
+    - echo 'end:coq.config'
```

Comment:
I wouldn't bother with those echos, they just make the script harder to read. The ones in the build template are just there on inertia from travis.